apps/srt-file-transmit.cpp

```diff
@@ -60,7 +60,8 @@ int main( int argc, char** argv )
         o_loglevel = { "ll", "loglevel" },
         o_buffer = {"b", "buffer" },
         o_verbose = {"v", "verbose" },
-        o_noflush = {"s", "skipflush" };
+        o_noflush = {"s", "skipflush" },
+        o_fullstats = {"f", "full-stats" };
 
```

Comment:
Isn't it better to simply add an option for stats `-s` (Skipflush should use uppercase S, as I remember) that can have a value or "shot" (default) and "full"? Or at least provide two possibilities and check that they are not used together.

Reply:
The idea is control the "clear" of the stats. So "full-stats" do not clear, and without it it clears. If you like I can search for another option.

Reply:
Hi @ethouris ,

You agree with this?
`-s|S:<stats-report-freq=0> - frequency of status report`

So, "-s" it's "shot" and "-S" it's "full".

Reply:
Just a small trouble is that the syntax supported by the options in srt-file-transmit is newer and an option can't have simultaneously no arguments and one argument. Meaning, the option will eat up one command line argument that follows it, if you declare it `ARG_ONE`. So, if you do `./srt-file-transmit -s filename.ts srt://:5000`, then `filename.ts` will be eaten up (although with `-s:0 filename.ts` will not, it has its argument already). Surely it won't be a problem if you place `-s` as the last one, or follow it directly by `--`. With `-s` option mind that it's already occupied by another option, but I think this application is fresh enough so that we can change it, especially that it's a "developer only option". You can simply remove the short version of it.

Reply:
Hi @ethouris ,

I like to modify the code as less as possible. I'll prepare another patch, without changing the current behaviour. Please, be on hold!

Reply:
Hi @ethouris ,

I pushed a simple workaround: it continues to use the "-f" parameter for selecting **total** counters instead of default **shot** counters. This simplifies a lot the code. However, I included the ARG_NONE in the `srt-file-transmit` tool for "-skipflush" & "-fullstats". I feel this resolve your case. Futhermore, I added two missing parameters in the help of `srt-live-transmit`.

I hope you agree, and this branch can be merged soon!
Regards! :smile: